csharp/Api/BUILD

```diff
@@ -0,0 +1,50 @@
+#
+# Copyright (C) 2022 Vaticle
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+package(default_visibility = ["//visibility:public"])
+
+
+load("//csharp:build_opts.bzl", "nullable_context", "target_frameworks", "targeting_packs")
+load("@rules_dotnet//dotnet:defs.bzl", "csharp_library")
+load("@vaticle_dependencies//tool/checkstyle:rules.bzl", "checkstyle_test")
+
+
+csharp_library(
+    name = "api",
+    srcs = glob(["*.cs", "*/*.cs", "*/*/*.cs"]),
```

Comment:
This can be `**/*.cs`, which is recursive.

Reply:
Thanks! I copied what Java does, and this style is more explicit for the architecture's understanding, I guess. I'd leave it like this, but I'm ready to refactor it if you want me to.